setup/install_gatling/30_post_install/99_setup_gatling_proxy.rb

```diff
@@ -0,0 +1,25 @@
+test_name 'Setup and configure gatling proxy machine' do
+  step 'install java, xauth' do
+    on metric, 'yum -y install java-1.8.0-openjdk java-1.8.0-openjdk-devel xauth'
+  end
+  step 'install scala build tool (sbt)' do
+    on metric, 'rpm -ivh http://dl.bintray.com/sbt/rpm/sbt-0.13.16.rpm'
+  end
+  step 'install rvm, bundler' do
+    on metric, 'gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3'
+    on metric, 'curl -sSL https://get.rvm.io | bash -s stable --ruby=2.2.5'
+    on metric, 'source /usr/local/rvm/scripts/rvm'
+    on metric, 'gem install bundler'
```

Comment:
Line 11 and 12 will happen in separate shells won't they?  So anything 11 does will be lost?  Probably 12 creating a new shell does the same thing and 11 can be removed.

Reply:
yeah. you are right. the source does nothing 🤷‍♂️ 
it might not be required at all.

Reply:
if gem works after i remove the source (which it should because there is only one ruby version installed; AND because it would have executed in a different shell/session anyway), then the source is spurious.